src/installer/windows/index.ts

```diff
@@ -11,17 +11,20 @@ import {Installation} from './installation'
 
 export class WindowsToolchainInstaller extends VerifyingToolchainInstaller<WindowsToolchainSnapshot> {
   private get vsRequirement() {
-    const reccommended = '10.0.17763'
+    const recommended = '10.0.19041'
     const current = os.release()
-    const version = semver.gte(current, reccommended) ? current : reccommended
-    const winsdk = semver.patch(version)
+    const version = semver.gte(current, recommended) ? current : recommended
+    const winsdkMajor = semver.lt(version, '10.0.22000')
+      ? semver.major(version)
+      : 11
+    const winsdkMinor = semver.patch(version)
```

Comment:
please extract the windows major version logic to a separate function, it will be easier to update in future.